src/Elmish/Hooks/UseEffect.js

```diff
@@ -0,0 +1,17 @@
+const React = require("react")
+
+class LifeCycles extends React.Component {
+  constructor(props) {
+    super(props)
+  }
```

Comment:
If you're just calling `super`, you can remove the whole constructor.

```suggestion
```

Reply:
Oh right 🤦🏻